doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -422,14 +422,153 @@ int main(void)
 </div>
 
 <div id="app-java" class="tab-pane fade" markdown="1" >
+```java
+
+import org.kaaproject.kaa.client.DesktopKaaPlatformContext;
+import org.kaaproject.kaa.client.Kaa;
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
+import org.kaaproject.kaa.client.configuration.base.ConfigurationListener;
+import org.kaaproject.kaa.client.configuration.base.SimpleConfigurationStorage;
+import org.kaaproject.kaa.client.logging.strategies.RecordCountLogUploadStrategy;
+import org.kaaproject.kaa.schema.sample.Configuration;
+import org.kaaproject.kaa.schema.sample.DataCollection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Class implement functionality for First Kaa application
+ *
+ * @author Maksym Liashenko
+ */
+public class FirstDemo {
```

Comment:
This should be either FirstKaaDemo or more meaningful class name (TemperatureSensorEmulator)